docs/src/templates/pages/styleguide/ui-components/index.hbs

```diff
@@ -22,19 +22,29 @@ Note. Only components that are part of the global platform should be added to th
 
 {{/markdown}}
     </div>
-    <div class="g-col g-span6">
+    <div class="g-col g-span4">
 {{#markdown}}
 
 <h3 class="sg-sectionHeading">Components</h3>
 
 - [Headers](headers/)
 - [Footers](footers/)
 - [Breadcrumbs](breadcrumbs)
-- [Cards](cards)
-- [Badges](Badges)
 - [Star Ratings](star-ratings)
 - [Listings](listings)
 - [Media Element](mediaElement)
+- [Icons](ui-icons)
+
+{{/markdown}}
+    </div>
+    <div class="g-col g-span4">
+{{#markdown}}
```

Comment:
Please could you indent these markdown tags and everything inside them so that it looks like they're inside the `div`s?

Reply:
Annoyingly it breaks the rendered HTML if you indent the content within the `markdown` sections — adding spaces converts the markup to a code block 🤦‍♂️ 

I _can_ indent the `markdown` tags and leave the content un-indented?